web/src/engine/osk/src/banner/suggestionBanner.ts

```diff
@@ -726,21 +747,23 @@ export class SuggestionBanner extends Banner {
     const emSizeStr = getComputedStyle(document.body).fontSize;
     const emSize    = getFontSizeStyle(emSizeStr).val;
 
-    const textStyle = getComputedStyle(this.options[0].container.firstChild as HTMLSpanElement);
+    const textElementStyle = getComputedStyle(this.options[0].container.firstChild as HTMLSpanElement);
 
     const targetWidth = this.width / SuggestionBanner.LONG_SUGGESTION_DISPLAY_LIMIT;
+    const height = this.height * .85; // .85 is a modifier seen in the CSS.
```

Comment:
Why do we have hard-coded scaling here? Wouldn't that have to change based on the ratio as well?

Reply:
I've extracted this to a constant at the top of the page.  Not claiming it's a perfect solution, but that should at least be more easily maintainable.